### PR TITLE
Moved the app to postgres using docker

### PR DESCRIPTION
**Pull request by Lifeissimple-zxc**
The migration requires running the app a bit differently so I have also added a README. The PR also features some minor change to JS (needed for client redirection after an OK registration).

### CODE REVIEW

File: static/scripts/app-register.js [hunk 1]
```diff
@@ -54,119 +54,120 @@ formbtn.addEventListener('click', async (e) => {
...
+        headers: {
+            'Content-Type': 'application/json;charset=utf-8'
+            },
+        body: jsonData
+        });
+
+    switch (datafetch.status) {
+        case 200:
+            console.log('successful registration');
+            // TODO @mike implement better comms to user
```
Comment (FirstDayAtWork) [inline static/scripts/app-register.js:79]: how? add some green banner with text like 'SUCCESS' ??

File: static/scripts/app-register.js [hunk 2]
```diff
@@ -54,119 +54,120 @@ formbtn.addEventListener('click', async (e) => {
...
+            return;
+        case 400:
+            console.log('bad request, try again');
+            // TODO @mike implement some communication to user
+            return;
+        case 500:
+            console.log('server error')
+            // TODO @mike implement some communication to user
+            return;
+        case 409:
```
Comment (FirstDayAtWork) [inline static/scripts/app-register.js:90]: I think we can combine 400, 500 & 409 codes into one IF statement with code from 92-107 lines + error message from server

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Arranging files by folders / labelling track files to simplify mxing in record box or another mixing program. 
+
+## Solution
+Tune Tracker (name is not final) is a web application that aims to solve this problem giving the users an opportunity to provide a spotify playlist URL to monitor for new tracks. Locating, purchasing and downloading of the music will be handled automatically.
+
+## Architecture
+
+TBD
+
+## Running the App Locally
```
Comment (FirstDayAtWork) [inline README.md:16]: so i should follow this guide first, yahs ?

File: static/scripts/app-register.js (hunk 1 above)
Comment (Lifeissimple-zxc) [reply static/scripts/app-register.js:79]: Yeah, some sort of UI "sugar" that will inform the clientele that the operations have succeeded.

File: README.md (hunk 3 above)
Comment (Lifeissimple-zxc) [reply README.md:16]: Yep, will require some installation, but should work! 👍

File: static/scripts/app-register.js (hunk 2 above)
Comment (Lifeissimple-zxc) [reply static/scripts/app-register.js:90]: Feel free to do it in a separate PR. Or you'd rather I take this task?

Comment (FirstDayAtWork) [reply static/scripts/app-register.js:90]: no i

**Review by FirstDayAtWork — APPROVED** [review]
so i test pr on my machine and now it works!

**Pull request merged**